modules/gdscript/gdscript_editor.cpp

```diff
@@ -1264,7 +1265,19 @@ static void _find_identifiers_in_base(const GDScriptCompletionIdentifier &p_base
 				}
 				return;
 			} break;
-			case GDScriptParser::DataType::ENUM:
+			case GDScriptParser::DataType::ENUM: {
+				String type_str = base_type.native_type;
+				StringName type = type_str.substr(0, type_str.find("."));
+				StringName type_enum = base_type.enum_type;
+
+				List<StringName> enum_values;
+				ClassDB::get_enum_constants(type, type_enum, &enum_values);
+				for (const StringName &E : enum_values) {
+					ScriptLanguage::CodeCompletionOption option(E, ScriptLanguage::CODE_COMPLETION_KIND_CONSTANT);
+					r_result.insert(option.display, option);
+				}
+			}
+				[[fallthrough]];
```

Comment:
Put after the bracket directly, cleaner

Reply:
I did this and the "Style checks via pre-commit" test didn't allow it, shall I put it back as it was?

Reply:
Sorry for missing this, yes, but odd too, I've seen it like that too